models/endpoints.models.js

```diff
@@ -0,0 +1,8 @@
+const fs = require('fs/promises')
+
+exports.getEndpointsData = () => {
+    return fs.readFile('./endpoints.json', 'utf8').then((data) => {
+        const endpoints = JSON.parse(data)
+        return endpoints
+	})
+}
```

Comment:
You don't really need a model for this endpoint, generally a model should deal with the database only and because you aren't dealing with a database, you could do this in the controller.